src/argilla_sdk/records/_dataset_records.py

```diff
@@ -338,36 +339,49 @@ def from_json(self, path: Union[Path, str]) -> "DatasetRecords":
             DatasetRecords: The DatasetRecords object created from the disk path.
 
         """
-        with open(path, "r") as f:
-            records = json.load(f)
-        self.update(records=[_dict_to_record(record) for record in records])
+        records = JsonIO._records_from_json(path=path)
+        self.update(records=records)
```

Comment:
Does make sense to return the loaded records instead?
```suggestion
        return self.update(records=records)
```